.circleci/config.yml

```diff
@@ -188,8 +188,6 @@ workflows:
           matrix:
             parameters:
               version:
-                - 12.22.10
-                - 14.19.0
```

Comment:
Double check that the consumers of this library have also upgraded. I know that CLI in particular were a bit behind

Reply:
Oh thanks, I was wondering which consumer would have such low node version, thanks!

Reply:
They were definitely planning on upgrading, but not sure if they found the time or not 🤞

Reply:
It looks like we only support 18: https://github.com/snyk/cli/blob/2872c6f78fe7b58207a7cd62aab3dcfd8addd98f/package.json#L24 – is that correct?